### DISCUSSION TITLE
Cable Trace - Modeling Fiber Modules / Path Split Error

### DISCUSSION THREAD

**Discussion by shepherdjay** [netbox-community/netbox#13737] [Q&A] [answered]
Running into an issue with a cable trace due to modular fiber trays/splices and looking for suggestions before I consider possibly modeling either an adapter that doesn't exist or similar.

So we have 1U and 2U modular cases that can take several types of fiber modules or copper modules. For our purposes we'll focus on a 12 strand lc fiber module which is most of our deployment. The 1U accepts 2 modules, and the 2U accepts 4 modules.

I've modeled this accurately in netbox in that there is a 1u parent device with module bays (A, B, C, D)

For the modules I've modeled the 12 strand fiber module with 6 LC ports on front. 1 rear splice port on back with six positions. 

Now, naturally if we install two of these modules at time of build we don't run two separate 12 strand fiber. Instead we run 24 strand fiber. I attempted to use multi-cable termination and created single cable with the two terminations to the rear on one end and two terminations to the rear on the other. However it shows a PATH split on a cable trace.

I suspect path split is because from the calculations perspective Rear port A1-6 is connected to both A1-6, and B1-6 in the link peers. Even though its actually a one-to-one mapping.

**Comment by DanSheps** [comment]
This is being worked on in #11079 (PR: #13337), TBH I would just wait.  :D

**Reply by shepherdjay** [reply]
And I thought I had done a pretty good issue search... It will be a bit until we perform the upgrade to the version this will likely be fixed in. Still need to run the new minor version through our stage tests but given this, I will likely just wait.

**Reply by shepherdjay** [reply]
@DanSheps -- removed mark as answer... only because I wasn't able to test the new PR since I think it introduced a bug #13930

**Comment by shepherdjay** [comment, marked as answer]
I'm going to convert this to an idea due to comments in #13930

**Reply by DanSheps** [reply]
What do you mean convert it to an idea?
